pkgs/applications/graphics/cewe-fotobuch/EULA.txt

```diff
@@ -0,0 +1,35 @@
+﻿Lizenz- und Nutzungsbedingungen „Mein CEWE FOTOBUCH“

```

Comment:
It's an unfree package, so I suppose we should somehow reference the terms&conditions. For DwarfFortress, we ship a copy of an email allowing us to distribute it too.